ergoCubSN000/calibrators/right_arm-calib.xml

```diff
@@ -30,7 +30,8 @@
 		<param name="startupPosThreshold">     2                 2              2           2          2             2              2               90      90      90      90      90      90     </param>
 	</group>
 
-	<param name="CALIB_ORDER">  (4 5 6) (2) (0) (1) (3) (7 8 9 10) (11 12) </param> 
+<!--	<param name="CALIB_ORDER"> (4 5 6)  (2) (0) (1) (3) (7 8) (11 12) </param> --> 
+	<param name="CALIB_ORDER"> (3) (2) (0) (1) (7 8) (11 12) </param>
```

Comment:
Something we need to review.
cc @Nicogene @sgiraz @davidetome @AntonioConsilvio @mfussi66.

Reply:
Here I think the joints of the wrist has been disabled because a motor burnt